community/front-end/ofe/website/ghpcfe/cluster_manager/utils.py

```diff
@@ -222,3 +223,50 @@ def run_terraform(target_dir, command, arguments=None, extra_env=None):
             )
 
     return (log_out_fn, log_err_fn)
+
+def run_packer(target_dir, command, arguments=None, extra_env=None):
```

Comment:
Default `arguments` to `=[]` and `extra_env` to `={}`, unless they can be evaluated to False. I would add a docstring too

Reply:
That's done down in the function.